src/main/resources/static/js/common.js

```diff
@@ -322,8 +322,7 @@ function executeIfLdapIsDeactivated() {
 }
 
 function downloadSubscriptions(subscriptionData, subscriptionNames) {
-    var foundSubscriptions = subscriptionData.foundSubscriptions;
```

Comment:
Ensure that the back end does not return the json structure that was before {"notFoundSubscriptions":[], "foundSubscriptions":[]} But should now be ["subscription1", "subscription2",...]

Reply:
Tested it manually and it works.